.github/workflows/ansible-test-integration.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Integration
+on:
+  push:
+    paths:
+    - 'plugins/**'
+    - 'tests/**'
+    - '.github/workflows/ansible-test-integration.yml'
+  pull_request:
+    paths:
+    - 'plugins/**'
+    - 'tests/**'
+    - '.github/workflows/ansible-test-integration.yml'
+
+jobs:
+  pg-14:
+    uses: data-bene/ansible-collection-gh-workflow/.github/workflows/workflow.yml@release/v1
+    with:
+      docker-image: '[ "ubuntu2004" ]'
```

Comment:
How will this work if you need different containers for different ansible-core versions? Or can you only pick containers that are supported by *all* ansible-core versions you want to test against?

Also where do you configure which ansible-core versions to test against?

Reply:
by default it'll run on all supported dockers. Here I limit it to a subset, we can add more in the list. (see also `docker-include` for special cases)
it's possible to add:
```
with:
  ansible-core-version: '[ "stable-2.9", "stable-2.12" ]'
```

But if need special cases:
```
with:
  docker-include:
  - ansible-core-version: stable-2.9
    docker-image: fedora30         # python 3.7
    python-version: 3.8
```


Doc about input in this area is here : https://github.com/Data-Bene/ansible-test-versions-gh-action#inputs

Reply:
But then you need to include the workflow once for every stable branch + docker container combination, right?

Reply:
well here it's just about the GHA matrix: it's a cross product of vars, then it "excludes" then it "includes". For testing special case go with 'include' if it's just some special case to remove from built matrix, go with 'excludes', I believe it is pretty clear in the versions.yml file I mentioned earlier.